.gitignore

```
@@ -2,6 +2,11 @@ dist
 packages/*/lib
 packages/apps/*/lib
 
+
+packages/cli/policyIAM.json
+packages/cli/config/policyIAM.json
+packages/cli/service-account-file.json
```

```suggestion
policyIAM.json
service-account-file.json
```

Não gasta especificar o path aí também, em qualquer folder esses arquivos devem ser ignorados...
Talvez `servieAccountKey.json` (acho que key é melhor que file aqui, e cammel case porque o outro filename também está em cammel case 🙃 ) ?